src/components/header-menu/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HeaderMenuItem from '../header-menu-item';
+import { HeaderMenuProps } from './types';
+import LogoutButton from '../logout-button';
+import './header-menu.css';
```

Comment:
Предлагаю импортировать объект со стилями

```suggestion
import css from './header-menu.css';
```